components/ZoomOutMap.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
+  import geojsonExtent from "@mapbox/geojson-extent";
+  import { map } from "../stores.js";
+  import icon from "../assets/zoom_out_map.svg";
+
+  export let boundaryGeojson;
+
+  function recenter() {
+    $map.fitBounds(geojsonExtent(boundaryGeojson), {
+      padding: 20,
+      animate: true,
+      duration: 500,
+    });
+  }
+</script>
+
+<button type="button" title="Zoom to show entire boundary" on:click={recenter}>
```

Comment:
I tried a few variations of https://carbon-components-svelte.onrender.com/components/Button, but very obnoxiously they can't really take an SVG file as input; they need it wrapped as a svelte component. The styling also didn't necessarily seem awesome.